.github/workflows/test.yml

```diff
@@ -3,6 +3,9 @@ name: Test
 
 on: workflow_dispatch
 
+env:
+  INFO_LOG_FOLDER: 'eve-info.tar.gz'
```

Comment:
Seems we can save ourselves trouble specifying anything about info file in test.yml if we just hard-code output file name to `eve-info.tar.gz`

Reply:
done